index.html

```diff
@@ -0,0 +1,207 @@
+          <li><a href="#" rel="noreferrer noopener">Contact</a></li>
+        </ul>
+      </nav>
+      <ul class="hum">
+        <li class="hum"><i class="fas fa-bars"></i></li>
+      </ul>
+      <ul class="humb mobile-menu mobile-menu-toggle">
+        <li class="hum"><i class="fas fa-bars"></i></li>
+      </ul>
+      <div class="menu-overlay hide-menu">
+        <ul class="mobile-menu-toggle menu-close">
+          <li><i class="fas fa-times"></i></li>
+        </ul>
+        <ul class="mobile-menu-items mobile-menu-toggle">
+          <li><a href="#" rel="noreferrer noopener">Home</a></li>
+          <li><a href="about.html" rel="noreferrer noopener">About</a></li>
+          <li><a href="#" rel="noreferrer noopener">Courses</a></li>
+          <li><a href="#" rel="noreferrer noopener">Contact</a></li>
+        </ul>
+      </div>
+    </header>
+    <section class="home">
+      <h2>Welcome for Online Course Website 2021</h2>
+      <p>
+        The world has changed and education has become easy to obtain, so do not
+        worry, you will start a fantastic journey from which you will gain
+        knowledge and practical experience.
+      </p>
+    </section>
+    <section class="main-program">
+      <h1>Main Program</h1>
+      <p>Choose from List</p>
+      <div class="main-pro">
+        <div class="main-box">
+          <div class="border-box">
+            <i class="fas fa-laptop-code"></i>
+            <h3>Full Stack</h3>
+            <p>
+              Learn full stack web developer with html, css, javascript and
+              ruby.
+            </p>
+          </div>
+          <div class="border-box">
+            <i class="fa fa-bar-chart" aria-hidden="true"></i>
+            <h3>Data Science</h3>
+            <p>
+              Learn data science and do expert for analysis and make decision.
+            </p>
+          </div>
+          <div class="border-box">
+            <i class="fas fa-database"></i>
+            <h3>DataBase</h3>
+            <p>
+              Build strong database working with server and all required from
+              web or data science.
+            </p>
+          </div>
+          <div class="border-box">
+            <i class="fas fa-user-graduate"></i>
+            <h3>Scholarship</h3>
+            <p>
+              Scholarship helps you to achieve your goals and many scholarship
+              here you can choose from them.
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="learn-more">
+        <a class="learn" href="#" rel="noreferrer noopener">Learn More</a>
+        <a class="visit" href="#" rel="noreferrer noopener">Vist Courses</a>
+      </div>
+    </section>
+    <section class="feature-speak">
+      <!--<h3 class="feature">Featured Speakers</h3>
+      <br />
+      <div class="feature-s">
+        <img src="/src-image/drzekri.jpg" alt="drzekri" />
+        <h3 class="pers-name">Abdel Rahman</h3>
```

Comment:
## Live version

1. Kindly make use of GitHub Pages to deploy the web pages.
1. The project should be deployed properly, all pages, links, and images work as expected.
1. Add a link to the deployed project is present in the README.